src/main/java/ro/sapientia/furniture/controller/OrderController.java

```diff
@@ -26,27 +26,27 @@ public OrderController(final OrderService orderService) {
 		this.orderService = orderService;
 	}
 	
-	@GetMapping
+	@GetMapping("/all")
 	public ResponseEntity<List<Order>> findAllOrder(){
 		return new ResponseEntity<>(this.orderService.findAllOrder(),HttpStatus.OK);
 	}
 	
-	@GetMapping("/{id}")
+	@GetMapping("/find/{id}")
 	public ResponseEntity<Order> findOrderById(@PathVariable("id")final Long id) {
 		return new ResponseEntity<>(this.orderService.findeOrderById(id),HttpStatus.OK);
 	}
 	
-	@PostMapping
+	@PostMapping("/add")
 	public ResponseEntity<Order> create(@RequestBody final Order order) {
 		return new ResponseEntity<>(this.orderService.create(order),HttpStatus.OK);
 	}
 	
-	@PutMapping
+	@PutMapping("/update")
```

Comment:
It's okay tin his way as well, but as we discussed to use the same annotations as a team, please use PostMapping